src/Screens/Gameplay/Components/Coins/CoinsTab.jsx

```diff
@@ -18,9 +17,9 @@ export default function CoinsTab() {
 			{texts.gameplay.coins.coinsTab.coinsAmountPt1}
 			XX
 			{texts.gameplay.coins.coinsTab.coinsAmountPt2}
-			<GeneralButton id='getMoreCoinsButton' onButtonClick={getMoreButtonClickHandler}>
+			<div className='generalButton' id='getMoreCoinsButton' onClick={getMoreButtonClickHandler}>
```

Comment:
this button wasn't uppercased, please keep it as it were.
can be done by adding `text-transform: none; border: 0px;` to it's id.